src/components/tile/index.js

```diff
@@ -0,0 +1,3 @@
+import Tile from './tile.component';
```

Comment:
I see that we use shorter syntax in index.js files so in this case it should be
`export { default } from './tile.component'`
Can you change it to keep syntax clean please?

Reply:
Done, will wait for QA to finish before requesting re-reviews though, in case any bugs come up.